client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -68,7 +68,16 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 					this.props.isSaveRequestSuccessful &&
 					( this.props.isJetpackSaveRequestSuccessful || ! this.props.jetpackSettingsUISupported )
 				) {
-					this.props.successNotice( this.props.translate( 'Settings saved!' ), {
+					let successNoticeText = this.props.translate( 'Settings saved!' );
+
+					if (
+						this.props.dirtyFields.includes( 'podcasting_category_id' ) &&
+						this.props.fields.podcasting_category_id === '0'
+					) {
```

Comment:
This feels dirty... :-)

Hardcoding the special notice text here, in a general site settings component, with

1) knowledge of the podcasting feature, and
2) knowledge of how podcasting feature fields are set to be disabled

I'm honestly not sure that the slight improved UX is worth the maintenance cost of special-casing like this.